sbin/build.sh

```diff
@@ -1067,6 +1067,25 @@ generateSBoM() {
   # Add CycloneDX versions
   addCycloneDXVersions
 
+  local formulaName="formula_temurin_build_script_1.0_jdk21u"
```

Comment:
We will need to construct these names based on the given builds VERSION etc...
I suggest something like:
```
local formulaName="formula_temurin_build_script_${fullVer}"
```
fullVer from https://github.com/adoptium/temurin-build/blob/f4bcbeb80b2dc0a624721f09a91c925e6df2bc90/sbin/build.sh#L1010C9-L1010C16